### ISSUE TITLE
password change verification by forcing to enter it twice and automatically comparing both

### ISSUE THREAD

**Issue by Knaui** [projectsend/projectsend#855]
when a user or client is changing a password, there is no second field for repeated password input.
this can lead to a misspelled password and having to reset the password again.
I know that there is a reveal function but I had issues with users because nobody seems to view the password in clear text before hitting "ok".

**Comment by ignacionelson** [comment]
Hi! It was hard to decide between repeating password fields or a single one with a reveal button.
It would not be too complicated to add a second field to your installation but I wouldn't like to apply this modification as part of PS itself. UX standards go against it lately.
Here's a nice article about this: https://cxl.com/blog/password-ux/
Maybe a better solution would be to make the reveal button more prominent?

**Comment by kkplein** [comment]
Agreed Ignacio, please keep it as it is. (perhaps add your own suggestion) :-)

**Comment by Knaui** [comment]
Ah ok i understand.
No biggy.
Didn't know that both functions together, double field and reveal button, are difficult to implement.

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
